src/ray/util/event_test.cc

```diff
@@ -205,6 +205,13 @@ TEST(EVENT_TEST, TEST_BASIC) {
   CheckEventDetail(result[3], "", "", "", "GCS", "FATAL", "", "");
 }
 
+TEST(EVENT_TEST, TEST_VARDIC_TEMPLATE) {
+  TestEventReporter::event_list.clear();
+  ray::EventManager::Instance().ClearReporters();
+  RAY_CUSTOM_EVENT("int", 123, "string", "string", "double", 1.24f);
```

Comment:
Can we avoid adding a new API? 
```suggestion
  RAY_EVENT(ERROR,"label1")
            .WithField("int", 123).WithField("string", "string").WithField("double", 1.24f) << "message";
```